gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ActiveClusterMonitor.java

```diff
@@ -62,7 +62,8 @@ public ActiveClusterMonitor(
     public void start()
     {
         log.info("Running cluster monitor with connection task delay of %s", taskDelay);
-        scheduledExecutor.scheduleAtFixedRate(() -> {
+        @SuppressWarnings("unused")
+        var unused = scheduledExecutor.scheduleAtFixedRate(() -> {
```

Comment:
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-var

Reply:
Also, we can do `Future<?> _ = ...` instead of suppressing the unused warning. This is a modern Java way of saying "I don't care about this value".